platform/android/export/export.cpp

```diff
@@ -1155,6 +1164,7 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 
 	virtual void get_export_options(List<ExportOption> *r_options) {
 
+		r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "graphics/arcore"), false));
```

Comment:
@BastiaanOlij Instead of following this approach, you should add ARCore as another XR mode entry such as the XR mode selection drop-down will show: `Regular`, `Oculus Mobile VR`, `AR Core`.

This also affects the logic to update the manifest file, as you no longer need to add another `meta-data` for ARCore, but you can just reuse the `xr_mode_metadata` one, and set the value accordingly if ARCore is the selected XR mode.

Reply:
That is a pretty good idea, I'll do that this week